wikidata/entity.py

```diff
@@ -237,6 +237,10 @@ def attributes(self) -> Mapping[str, object]:
     def load(self) -> None:
         url = './wiki/Special:EntityData/{}.json'.format(self.id)
         result = self.client.request(url)
+        if result is None:
+            self.data = None
```

Comment:
The existing code assumes if `self.data is None` it means entity data is not loaded *yet*, not non-existent.  So this code leads any `Entity` objects referring to non-existent entities to send a HTTP request again and again for every property/method call towards them.

Instead, we need three distinct types for three different cases:

1. `None`: Not-loaded yet
2. `Mapping[str, object]`: Existent and loaded
3. (to be chosen): Non-existent